source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/layout/StretchableInputLayout.java

```diff
@@ -1,13 +1,24 @@
 package io.adaptivecards.renderer.layout;
 
 import android.content.Context;
+import android.graphics.PorterDuff;
 import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
```

Comment:
This one for eg; Could you update with the copyright header here while you are making changes to the file? Thanks